dashboard/app/assets/javascripts/soundEffects.js

```diff
@@ -227,24 +227,35 @@ Sound.prototype.newPlayableBufferSource = function(buffer, options) {
   var newSound = this.audioContext.createBufferSource();
```

Comment:
i'm not quite sure of the relationship here, but it is odd for something in apps to depend on something in dashboard?